resources/enrich_sci_crunch_csv.py

```diff
@@ -0,0 +1,158 @@
+    org_name = row['Parent Org Name']
+    filled_ror = row['ROR and other mappings']
+    org_info = {'proposed': '', 'ror_id': ''}
+
+    if (not empty_cell(org_name)) and (not ('ror.org' in filled_ror)):
+        if org_name in ror_cache:
+            org_info = ror_cache[org_name]
+        else:
+            ror_info = best_ror(org_name)
+            if ror_info:
+                org_info = ror_info
+            ror_cache[org_name] = org_info
+            time.sleep(0.05)
+
+    return org_info
+
+
+def full_enhance(args):
+    ror_cache = {}
+    out_file = os.path.join(os.path.dirname(args.input), 'scicrunch_working_file_enriched.csv')
+
+    with open(args.input, 'r') as csv_file:
+        csv_reader = csv.DictReader(csv_file)
+        headers = csv_reader.fieldnames
+
+        last_column = 39
+        fixed_headers = headers[:last_column + 1]
+        new_headers = ['proposed_name', 'proposed_ror_id']
+        processed_rows = 0
+
+        with open(out_file, 'w', newline='') as csv_out:
+            csv_writer = csv.writer(csv_out)
+            csv_writer.writerow(fixed_headers + new_headers)
+
+            for row in csv_reader:
+                processed_rows += 1
+                if processed_rows % 100 == 0:
+                    print(f"processed {processed_rows} rows")
+
+                org_info = proposed_ror_info(row, ror_cache)
+                new_row = [row[key] for key in headers[:last_column + 1]] + [org_info['proposed'], org_info['ror_id']]
+                csv_writer.writerow(new_row)
+
+
+def minimal_info(args):
+    ror_cache = {}
+
+    with open(args.input, 'r') as csv_file:
+        csv_reader = csv.DictReader(csv_file)
+
+        processed_rows = 0
+        out_file = os.path.join(os.path.dirname(args.input), 'scicrunch_working_file_minimal.csv')
+        with open(out_file, 'w', newline='') as csv_out:
+            csv_writer = csv.writer(csv_out)
+            csv_writer.writerow(['software_name', 'github_slug', 'ror_id', 'org_name', 'extraction_methods'])
+
+            for row in csv_reader:
+                processed_rows += 1
+                if processed_rows % 100 == 0:
+                    print(f"processed {processed_rows} rows")
+
+                proposed_info = proposed_ror_info(row, ror_cache)
+                if (not ('ror.org' in row['ROR and other mappings'])) and (not proposed_info['ror_id']):
+                    continue
+
+                ror_id = row['ROR and other mappings'] if ('ror.org' in row['ROR and other mappings']) else \
+                    proposed_info['ror_id']
+                org_name = row['Parent Org Name']
+                sfw_name = row['Resource_Name']
+                extraction_methods = 'human_curated' if ('ror.org' in row['ROR and other mappings']) else 'by_name'
+                new_row = [sfw_name, github_slug(row), ror_id, org_name, extraction_methods]
+                csv_writer.writerow(new_row)
+
+
+parser = argparse.ArgumentParser()
```

Comment:
You've got a couple potentially reusable pieces of code here to do with retrieving info from the ROR API. It may make sense to put this code in a 

```
if __name__ == "__main__":
  parser = argparse... etc
```

block, so that it will only execute if someone is running this script from the command line (as opposed to importing its functions elsewhere)